docs/standards/nft-2.0/LSP8-Identifiable-Digital-Asset.md

```diff
@@ -13,83 +13,98 @@ sidebar_position: 4
 
 ## Introduction
 
-When it comes to Non Fungible Tokens (NFTs), one of the key characteristics that make them unique is their **metadata.**
+Non-Fungible assets represented mainly by **[ERC721](https://eips.ethereum.org/EIPS/eip-721)** and **[ERC1155](https://eips.ethereum.org/EIPS/eip-1155)** have a lot of limitations in terms of metadata, secure transfers, asset representation, and asset interaction. This is causing major problems for users seeking, **full control** over which assets they accept or not, **more complex NFTs**, and a **simple user experience** while creating, buying, and exchanging assets.
 
-In their current state, such assets are created as ERC721 tokens. However, their limitations are following:
+**[LSP8-IdentifiableDigitalAsset](#)** is the standard that aims to solve all problems mentioned above by allowing, more secure transfers via **force bool**, more asset metadata **via [LSP4-DigitalAssetMetadata](./LSP4-Digital-Asset-Metadata.md)**, more asset representation **via bytes32 tokenIds**, and more interaction between the asset contract and the asset *sender/recipient* **via token hooks**.
 
-#### 1. The metadata attached to the NFT (represented with `tokenURI`) refers to a file.
+![LSP8IdentifiableDigitalAsset features Introduction](/img/standards/lsp8-intro.jpeg)
 
-The reference raises several questions around where the token's metadata is stored. Is a central service used to store the metadata? What about if such a service goes down?
+## What does this Standard represent ?
 
-![ERC721 Metadata Security](/img/erc721-metadata-security.png)
+### Specification
 
-#### 2. The only data that can be attached to an NFT is a `name` and `symbol`.
+**[LSP8-IdentifiableDigitalAsset](#)** is a standard that aims to describe _non-fungible_ assets. The term _Non-fungible_ means that each asset is **unique and different**. They are distinguishable from each other and therefore **not interchangeable**.
 
-Each NFT has its own characteristics and specific details. The singularity is what makes each NFT unique. But how can we attach more information specific to an NFT? Or **how can we query one precise information** specific to an NFT without having to retrieve and search through the entire metadata?
+This standard was based on **[ERC721](https://eips.ethereum.org/EIPS/eip-20)** and **[ERC1155](https://eips.ethereum.org/EIPS/eip-777)**. It got enhanced by **unifying function names**, adding more functions (**batch transfers**), and lots of **new features** mentioned below.
```

Comment:
Im not sure what 'unifying function names' is? 

We can say what the new features are? e.g '...and other new features such as x, y and z discussed further below.'

We don't really need these docs to be pitch for why the LSP standards are good, just a factual description of what the benefits are and what they allow us to do with examples